bciers/apps/administration/tests/components/contacts/ContactForm.test.tsx

```diff
@@ -236,6 +243,7 @@ describe("ContactForm component", () => {
               existing_bciers_user: false,
               first_name: "John",
               last_name: "Doe",
+              places_assigned: [],
```

Comment:
The `in` ninja schema doesn't have places_assigned, so we're never going to do anything with this. Problem that it's still being sent? I would have though `omitExtraData` would have pulled it out, but apparently not